cypress/e2e/ArticleViewing/article_content.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+const { url, articlePage, textInPages } = require('../../support/selectors')
+
+describe('Make sure BBC.com loads properlly', () => {
+    beforeEach(() => {
+        cy.visit(url.articleChosenToTest);
+    });
+
+    it('Test Case 20. Verify article content', () => {
+        cy.title().should('include',textInPages.articleTitlePiece);
+        cy.get('time').should('be.visible');
+        cy.get(articlePage.videoPlayer).should('be.visible');
+        cy.get(articlePage.imgOfArthur).should('be.visible');
+    });
+
+    it('Test Case 21. Article share button test', () => {
+        cy.get(articlePage.shareButton).click({force: true});
+        cy.get(articlePage.copyButton).click({force: true});
+        cy.get(articlePage.shareDropbox).find(articlePage.copiedLink).should('exist');
+        cy.get(articlePage.facebookButton).should('be.visible');
+        cy.get(articlePage.twitterButton).should('be.visible');
+        cy.get(articlePage.emailButton).should('be.visible');
```

Comment:
o tu testuoji ar jie veikia, ar tiesiog ar jie yra matomi ?

Reply:
Su linkais (pvz facebook, twitter) nesuradau kaip priversti juos atsidaryti tame paciame tab'e del to vienintelis ka sugebejau tai yra isitikinti ar jie matomi. Su copy to clipboard nesugebejau paziureti i clipboarda ka issaugojo nes pasirodo tai yra didelis security issue.

Reply:
cy.get(element).invoke('removeAttr', 'target').click()

cy.get(element).invoke('attr', 'target', ' _self').click()

bandei su kazkuriuo is situ tai daryti ?

Reply:
taip. Neveikia